### PR TITLE
fix: vulnerable versions

### PR DESCRIPTION
**Pull request by yihong0618**
fix some vulnerable versions by `Dependabot alerts`

### CODE REVIEW

File: dockerfiles/requirements-py37.txt [hunk 1]
```diff
@@ -72,8 +72,8 @@ netCDF4==1.5.8
 nltk==3.7
 numba==0.55.2
 numexpr==2.8.1
-numpy==1.21.6
-oauthlib==3.2.0
+numpy==1.22
```
Comment (beeender) [inline dockerfiles/requirements-py37.txt:75]: numpy `1.22` doesn't seem to support python3.7
https://pypi.org/project/numpy/1.22.0/

File: dockerfiles/requirements-py37.txt [hunk 2]
```diff
@@ -84,7 +84,7 @@ pdfminer.six==20220524
 Pillow==9.1.1
 portend==3.1.0
 preshed==3.0.6
-protobuf==3.19.4
+protobuf==3.19.5
```
Comment (beeender) [inline dockerfiles/requirements-py37.txt:87]: why not 3.196?
https://pypi.org/project/protobuf/3.19.6/

File: dockerfiles/requirements-py39.txt [hunk 3]
```diff
@@ -85,7 +85,7 @@ pdfminer.six==20220524
 Pillow==9.1.1
 portend==3.1.0
 preshed==3.0.6
-protobuf==3.19.4
+protobuf==3.19.5
```
Comment (beeender) [inline dockerfiles/requirements-py39.txt:88]: ditto

File: dockerfiles/requirements-py39.txt [hunk 4]
```diff
@@ -73,8 +73,8 @@ netCDF4==1.5.8
 nltk==3.7
 numba==0.55.2
 numexpr==2.8.1
-numpy==1.22.1
-oauthlib==3.2.0
+numpy==1.22
```
Comment (beeender) [inline dockerfiles/requirements-py39.txt:76]: just go to the latest
https://pypi.org/project/numpy/1.23.3/

**Comment by yihong0618** [comment]
copy that will fix

File: dockerfiles/requirements-py37.txt (hunk 1 above)
Comment (yihong0618) [reply dockerfiles/requirements-py37.txt:75]: opps.....

**Comment by yihong0618** [comment]
for numpy python3.7 we can ignore the alert: see https://github.com/zauberzeug/nicegui/issues/67

**Review by beeender — APPROVED** [review]

**Pull request merged**